lib/sso_users_api/manager_job.rb

```diff
@@ -1,30 +1,16 @@
-require 'active_job'
+require 'sidekiq'
 
 module SsoUsersApi
-  class ManagerJob  < ActiveJob::Base
-    queue_as :sso_users_api
+  class ManagerJob
+    include Sidekiq::Worker
 
-    def self.delay_amount
-      2
-    end
+    sidekiq_options queue: :high_priority, retry: 0
 
-    def perform(id, class_name, count = 0, options = {})
+    def perform(id, class_name, options = {})
       user = class_name.constantize.find(id)
       SsoUsersApi::Manager.new(user).call
-      begin
-        options[:on_success_call_back_job_name].constantize.perform_later(id) if options[:on_success_call_back_job_name].present?
-      rescue StandardError => e
-        NfgRestClient::Logger.error("Failed to execute: #{options[:on_success_call_back_job_name]}, error: #{e.message}")
```

Comment:
This string hasn't been logged during the past thirty days. I think it's safe to rely on Sidekiq to raise the exception and send it to Honeybadger in the unlikely event that we would even reach this point. We would also be able to see the job on the 'dead' section of the sidekiq_web ui.